py/picca/tests/delta_extraction/astronomical_object_tests.py

```diff
@@ -1668,14 +1668,14 @@ def test_forest_set_class_variables(self):
             3.6463025, 3.6513025, 3.6563025, 3.6613025, 3.6663025, 3.6713025,
             3.6763025, 3.6813025, 3.6863025, 3.6913025, 3.6963025, 3.7013025,
             3.7063025, 3.7113025, 3.7163025, 3.7213025, 3.7263025, 3.7313025,
-            3.7363025, 3.7413025
+            3.7363025, 3.7413025,
         ])
         self.assertTrue(np.allclose(Forest.log_lambda_grid, log_lambda_grid))
 
         log_lambda_rest_frame_grid = np.array([
-            3.01953334, 3.02453334, 3.02953334, 3.03453334, 3.03953334,
-            3.04453334, 3.04953334, 3.05453334, 3.05953334, 3.06453334,
-            3.06953334, 3.07453334
+            3.01703334, 3.02203334, 3.02703334, 3.03203334, 3.03703334,
```

Comment:
this is a change by exactly 0.0025, i.e. one half pixel, did we want to change the definition? see other file